### ISSUE TITLE
Share link: Download button don't work in gallery view

### ISSUE THREAD

**Issue by michaelstingl** [owncloud/core#23821]
<img width="909" alt="owncloud" src="https://cloud.githubusercontent.com/assets/214010/14311781/64781906-fbe9-11e5-8cf7-61602e45f6be.png">

<img width="914" alt="owncloud_und_new_issue_ _owncloud_core" src="https://cloud.githubusercontent.com/assets/214010/14311780/6477f25a-fbe9-11e5-8612-8f3b184ad649.png">

### Steps to reproduce
1. Open link share with images and switch to gallery view
2. Click Download button on the top right

### Expected behaviour
A *.tar file should be downloaded

### Actual behaviour
Error message:
```
File not found
The specified document has not been found on the server.
```

### Server configuration

Shared hosting

**ownCloud version:** (see ownCloud admin page)
9.0.0

**Updated from an older ownCloud or fresh install:**
fresh install

**Where did you install ownCloud from:**
[setup-owncloud.php](https://owncloud.org/install/#instructions-server)

**Signing status (ownCloud 9.0 and above):**

```
No errors have been found.
```

**List of activated apps:**

```
$> php ./occ app:list
Enabled:
  - activity: 2.2.1
  - comments: 0.2
  - dav: 0.1.5
  - federatedfilesharing: 0.1.0
  - federation: 0.0.4
  - files: 1.4.4
  - files_pdfviewer: 0.8
  - files_sharing: 0.9.1
  - files_texteditor: 2.1
  - files_trashbin: 0.8.0
  - files_versions: 1.2.0
  - files_videoplayer: 0.9.8
  - firstrunwizard: 1.1
  - gallery: 14.5.0
  - notifications: 0.2.3
  - provisioning_api: 0.4.1
  - systemtags: 0.2
  - templateeditor: 0.1
  - updatenotification: 0.1.0
Disabled:
  - encryption
  - external
  - files_external
  - user_external
  - user_ldap
```

**The content of config/config.php:**

```
$> php ./occ config:list system
{
    "system": {
        "instanceid": "ocfvtj3oyv5q",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "mst.works"
        ],
        "datadirectory": "\/var\/www\/virtual\/multimst\/html\/oc\/data",
        "overwrite.cli.url": "http:\/\/mst.works\/oc",
        "dbtype": "sqlite3",
        "version": "9.0.0.19",
        "logtimezone": "UTC",
        "installed": true
    }
}
```

**Are you using external storage, if yes which one:** local/smb/sftp/...
no

**Are you using encryption:** yes/no
no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
no

### Client configuration
**Browser:**
Chrome 49

**Operating system:**
Mac OS X 10.11.4

**Comment by MorrisJobke** [comment]
cc @oparoz @rullzer @schiesbn

**Comment by oparoz** [comment]
Has been fixed and will be available in 9.0.1

**Comment by MorrisJobke** [comment]
> Has been fixed and will be available in 9.0.1

Seems to be https://github.com/owncloud/gallery/issues/554 -@oparoz correct me if I'm wrong.

**Comment by oparoz** [comment]
That's correct, thank you @MorrisJobke
